test/unit/federation_server_test.js

```diff
@@ -227,5 +227,98 @@ FEDERATION_SERVER="https://api.stellar.org/federation"
           .then(() => tempServer.close());
       });
     });
+
+  });
+
+  describe('FederationServer times out when response lags and timeout set', function () {
+    it("resolveAddress times out", function (done) {
+      StellarSdk.Config.setTimeout(100);
```

Comment:
We should reset it to default value so it doesn't influence other tests. Maybe in `afterEach`? Would be great to have `Config.setDefaults()` or something like this, in case default values change in a future.